custom_components/lock_code_manager/__init__.py

```diff
@@ -219,7 +228,11 @@ async def async_setup_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> b
         serial_number=entry_id,
     )
 
-    await hass.config_entries.async_forward_entry_setups(config_entry, PLATFORMS)
+    config_entry.async_create_task(
```

Comment:
Thanks for the PR! Out of curiosity, why did you move this into a task? If I recall we shouldn't call the entry setup before the platforms are setup which is why we block on this? Idk it's been a while